src/bin/flamegraph.rs

```diff
@@ -82,7 +82,7 @@ struct Opt {
         short = 'c',
         long = "colors",
         default_value = defaults::COLORS,
-        possible_values = &["aqua","blue","green","hot","io","java","js","mem","orange","perl","purple","red","rust","wakeup","yellow"],
+        possible_values = &["aqua","blue","green","hot","io","java","js","mem","orange","perl","python","purple","red","rust","wakeup","yellow","gray"],
```

Comment:
Would you mind doing `gray` as a separate PR, just so we don't hold up landing one because of discussion on the other?

Reply:
Will do

Reply:
```suggestion
        possible_values = &["aqua","blue","green","hot","io","java","js","mem","orange","perl","python","purple","red","rust","wakeup","yellow"],
```